templates/master-cert-secret.yaml

```diff
@@ -0,0 +1,9 @@
+{{- $creationDate := .Values.web.configuration.masterCert.creationDate | default (now | date "2006-01-02T15:04:05 -0700" ) }}
```

Comment:
Actually, this should be required instead of filling in the current timestamp. We could attempt to pull the creation date out of an possibly existing secret but that I think that's more work than necessary.